### DISCUSSION TITLE
Regarding vercel's contact person

### DISCUSSION THREAD

**Discussion by TakayamaYoshimichi** [vercel/community#6119] [Account Issue] [answered]
### Question

I would like to use vercel for web app of client's company and the company has asked me to set up one contact person in charge in vercel. Is there a service that can set up such a person?

**Comment by swarnava** [comment, marked as answer]
Hi,

We have responded to you via email. The dedicated Account Manager / Customer Success Manager is available only on [Enterprise plan](https://vercel.com/pricing).
